src/_select.scss

```diff
@@ -0,0 +1,111 @@
+// Select
+// ===
+
+
+// Configurable Variables
+// ---
+//
+// 1. The same as default side paddings (but they're set in 1 )
+
+$select__height: if(variable-exists(forms__element-height), $forms__element-height, 50px) !default;
+$select__v-space: if(variable-exists(v-space), $v-space, 0.5rem) !default;
+$select__h-space: if(variable-exists(h-space), $h-space, 1rem) !default;
+$select__padding: if(variable-exists(forms__element-padding), $forms__element-padding, 10px 15px) !default;
+
+$select__border-color: if(variable-exists(border-color), $border-color, hsl(0, 0%, 10%)) !default;
+$select__border: if(variable-exists(border), $border, 1px solid $select__border-color) !default;
+$select__border-radius: if(variable-exists(forms__border-radius), $forms__border-radius, 0) !default;
+
+$select__background-color: if(variable-exists(forms__background-color), $forms__background-color, #fff) !default;
+$select__triangle-size: 7px !default;
+$select__triangle-position: 15px !default; // 1
+
+$select__font-family: if(variable-exists(forms__font-family), $forms__font-family, inherit) !default;
+$select__font-size: if(variable-exists(forms__font-size), $forms__font-size, inherit) !default;
+$select__line-height: 20px;
+
+
+// Select Root
+// ---
+//
+// 1. Absolutely position select element and triangle relative to the parent .c-select
+// 2. Vertical center the triangle
+// 3. Align select text with input text
+
+.c-select {
+    position: relative; // 1
+
+    display: flex;
+    box-sizing: border-box;
+    max-width: 100%;
+    height: $select__height;
+    padding: $select__padding;
+    border: $select__border;
+
+    border-radius: $select__border-radius;
```

Comment:
To continue the config vars issue, this should just be zero and you can change it on your project.